app/gulpfile.js

```diff
@@ -19,29 +23,36 @@ gulp.task('buildMap', function(){
     browserifyShare('Map');
 });
 
-gulp.task('watch', function() {
+gulp.task('watch-app', function() {
```

Comment:
Actually for first point, there are other files under `server/` which are worth watching. Keeping things as they are for now.